README.md

```diff
@@ -9,6 +18,124 @@ The package relies on packages available in the following repositories:
 - [UMD](https://repository.egi.eu/)
 - [EPEL](https://docs.fedoraproject.org/en-US/epel/)
 
+## Installing and using the UI
+
+Once the UI will be installed, you will need to set it up so to be able to
+interact with the resources available to a given
+[Virtual Organisation (VO)](https://ims.egi.eu/display/EGIG/Virtual+organisation).
+
+### Deploying the UI
+
+The UI is available as a package in the [UMD](https://go.egi.eu/umd) software
+distribution, but it will also require additional software and configuration.
+
+In order to help with deploying the UI, different solutions are possible:
+
+- Deploying the UI manually, using the packages available from
+  [UMD repositories](https://go.egi.eu/umd). You will need to install at least
+  the `ui` meta-package, with `voms-client`, the
```

Comment:
i think the voms-client is alrady included in the metapackage

Reply:
you are right, fixed!

Reply:
For the CA, I need to check/test as https://docs.egi.eu/providers/operations-manuals/howto01_using_igtf_ca_distribution/#installation refers to a repository different from UMD, so there is a need to align things.

Reply:
So in fact the CA repo is configured by the umd-release package. I will rephrase/adapt.

Reply:
Done, please have a look.